Dockerfile-fsfreeze-pause-ppc64le

```diff
@@ -0,0 +1,19 @@
+# Copyright 2018, 2019 the Velero contributors.
```

Comment:
This can be just 2019 rather than 2018, 2019.

Reply:
fixed